schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+    id INTEGER PRIMARY KEY,
+    name VARCHAR(255),
+    date_of_birth DATE,
+    escape_attempts INTEGER,
+    neutered BOOLEAN,
+    weight_kg DECIMAL(10, 2),
+    species VARCHAR(255)
```

Comment:
- [ ] Kindly modify the table using the function ALTER TABLE instead of modifying the table directly for best practices 👇🏽 

```sql
ALTER TABLE animals ADD species varchar(255);
```